lib/ChGovUk/Controllers/Company/Document.pm

```diff
@@ -60,7 +66,7 @@ sub document {
                         $next->($document_metadata_uri);
                     }
                     else {
-                        $delay->emit('error', 'document_metadata not found in JSON');
+                        $delay->emit('not_found', $document_metadata_uri);
```

Comment:
Same here, there is not document metadata so we can't pass in. Maybe worth logging something like document metadata not found and changing the variable in not_found to $message

Reply:
Good points. will resolve shortly.